### DISCUSSION TITLE
Provision -> Neo4j fails

### DISCUSSION THREAD

**Discussion by monojit18** [Sunbird-Ed/Community#139] [Installation] [answered]
I am getting the below error while running Provision -> Neo4j

Branch: release-4.10.1_RC1
Tag: release-4.10.1_RC1
Docs followed: http://docs.sunbird.org/latest/developer-docs/server-installation/

Build -> KnowledgePlatform -> Neo4j and ArtifactUpload -> KnowledgePlatform -> Neo4j - Both are successful
But Provision -> KnowledgePlatform -> Neo4j fails with below error; asking for a blob - neo4j-community-3.3.9-unix.tar.gz
Can you please help/suggest?

fatal: [localhost]: FAILED! => {
    "ansible_job_id": "567310079165.11815",
    "changed": true,
    "cmd": [
        "az",
        "storage",
        "blob",
        "download",
        "-c",
        "artifacts",
        "--name",
        "neo4j-community-3.3.9-unix.tar.gz",
        "-f",
        "/var/lib/jenkins/workspace/Provision/dev/KnowledgePlatform/Neo4j/ansible/artifacts/neo4j-community-3.3.9-unix.tar.gz"
    ],
    "delta": "0:00:01.009040",
    "end": "2022-09-08 13:43:04.114215",
    "finished": 1,
    "rc": 3,
    "start": "2022-09-08 13:43:03.105175"
}

STDERR:

ERROR: The specified blob does not exist.

**Comment by beepdot** [comment, marked as answer]
Please see comment - https://github.com/project-sunbird/sunbird-devops/blob/master/private_repo/ansible/inventory/dev/KnowledgePlatform/common.yml#L22
You need to upload the neo4j tar.gz file to azure storage under `artificats` container.

**Reply by monojit18** [reply]
Thanks @keshavprasadms 
Can you please tell me from where to get this file? I do not see this version anywhere in the Release section.
Earliest I see is 3.5.35 - https://neo4j.com/download-center/#community

Can you please provide me the link?
Thanks

**Reply by daveanvit** [reply]
https://go.neo4j.com/download-thanks.html?edition=community&release=3.3.9&flavour=unix
@monojit18

**Reply by monojit18** [reply]
Thanks @daveanvit for sharing this.

Just now found it from [here](https://community.neo4j.com/t5/neo4j-graph-platform/older-versions-download-archive/td-p/47149) on how to download specific versions.

Will try to use it and update this thread.

**Reply by monojit18** [reply]
@keshavprasadms and @daveanvit 
Thanks for the prompt responses and help....this worked for me.
Marking this as Answered.
